### DISCUSSION TITLE
Is it possible to change the favicon and browser tab title?

### DISCUSSION THREAD

**Discussion by erobwen** [keycloakify/keycloakify#356] [Q&A]
I tried changing certain elements in /public/index.html in the keycloakify-starter project, and I replaced /public/favicon.ico for my own file, but for some reason these changes do not influence my built theme that I am testing in a docker container. I still get "React App" with a react logotype. What am I missing?

BR /Robert R

**Comment by garronej** [comment]
Hi @erobwen,  
I don't know, I have two Keycloakify theme in production, one is here:  
https://github.com/codegouvfr/sill-web  
Prod: https://sill.code.gouv.fr/  

<img width="1205" alt="image" src="https://github.com/keycloakify/keycloakify/assets/6702424/e46c1764-e37b-4ce5-b454-f26e4e4a106d">


The other is here:  
https://github.com/InseeFrLab/onyxia-web  
https://datalab.sspcloud.fr/home  

<img width="1150" alt="image" src="https://github.com/keycloakify/keycloakify/assets/6702424/826d7b27-0050-4bf1-942b-422a4b07b120">  

As you can see, in both cases favicon and title work as expected. Maybe if you share the core I'll be able to spot what's up.
